File: Solution/每日一题/2024.9.8/8.c
char* clearDigits(char* s) {
    int n = strlen(s), k = 0;
    char *res = (char *)malloc(n + 1);
    for (int i = 0; i < n; i++) {
        if (isdigit(s[i])) {
            k--;
        } else {
            res[k++] = s[i];
        }
    }
    res[k] = '\0';
    return res;
}

File: Solution/每日一题/2024.9.8/8.rs
impl Solution {
    pub fn clear_digits(s: String) -> String {
        let mut res: Vec<char> = Vec::new();
        for c in s.chars() {
            if c.is_digit(10) {
                res.pop();
            } else {
                res.push(c);
            }
        }
        res.into_iter().collect()
    }
}

